tests/test_sqs.py

```diff
@@ -0,0 +1,47 @@
+import boto3
+import pytest
+from botocore.exceptions import ClientError
+from moto import mock_sqs
+
+from awd import sqs
+
+
+@mock_sqs
+def test_sqs_send_success(
+    sqs_client, dss_message_attributes_example, dss_message_body_example
+):
+    sqs = boto3.resource("sqs", region_name="us-east-1")
+    sqs_queue = sqs.create_queue(QueueName="mock-queue")
+    response = sqs_client.send(
+        sqs_queue.url, dss_message_attributes_example, dss_message_body_example
+    )
+    assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
+
+
+@mock_sqs
+def test_sqs_send_failure(
+    sqs_client, dss_message_attributes_example, dss_message_body_example
+):
+    with pytest.raises(ClientError):
+        response = sqs_client.send(
+            "non-existent", dss_message_attributes_example, dss_message_body_example
+        )
+        assert response["ResponseMetadata"]["HTTPStatusCode"] == 200
```

Comment:
Does it still return 200 as status code if there's an error? Just confirming.

Reply:
Oops, sloppy copy-paste on my part. No, the test succeeds after the exception is triggered so no 200 status code. Removed this line in the most recent commit